scripts/Interface/reduction_gui/reduction/scripter.py

```diff
@@ -314,10 +319,10 @@ def __init__(self, name="", facility=""):
         if HAS_MANTID:
             config = ConfigService.Instance()
             try:
-                head, tail = os.path.split(config.getUserFilename())
+                head, _tail = os.path.split(config.getUserFilename())
                 if os.path.isdir(head):
                     self._output_directory = head
-            except:
+            except (StopIteration, StandardError, Warning):
```

Comment:
@picatess Yes these are the three sub-trees to exception, however pylint doesn't seem to complain about writing them separately and yet covers all. Maybe this link could be useful: https://docs.python.org/2/library/exceptions.html#exception-hierarchy